zips/zip-0000.rst

```diff
@@ -229,13 +231,46 @@ For each new ZIP that comes in, the ZIP Editors SHOULD:
 * Read the ZIP to check if it is ready: sound and complete. The ideas
   must make technical sense, even if they don't seem likely to be
   accepted.
-* Check that the title accurately describes the content.
-* Ensure that the ZIP draft has been sent to the Zcash Community Forum
-  and as a PR to the ZIPs git repository [#zips-repo]_.
+* Check that the Title, Category, and other metadata fields accurately
+  describe the content.
+* Ensure that the ZIP draft has been submitted as a PR to the ZIPs git
+  repository [#zips-repo]_. In some cases it may also be appropriate
+  for it to be sent to the Zcash Community Forum.
 * Ensure that motivation and backward compatibility have been
   addressed, if applicable.
 * Check that the licensing terms are acceptable for ZIPs.
 
+For proposals to revise an existing ZIP, the ZIP Editors SHOULD:
+
+* Read the proposed modification to check if it is ready: sound,
+  complete, and non-disruptive to the ZIP's existing deployments if any.
+* If the proposed change affects consensus and has already been deployed,
+  it SHOULD be written as an "Update ZIP" describing the changes to be
+  made, rather than directly as a pull request modifying the existing
+  ZIP or the Zcash Protocol Specification. At the discretion of the ZIP
+  Editors, this might not be needed in some cases for compatible changes
+  or clarifications.
+* If a change is to be made to an existing ZIP, check that the Title,
+  Category, and other metadata fields still accurately describe the
+  modified content.
+* Ensure that the update has been submitted as a PR to the ZIPs git
+  repository [#zips-repo]_. In some cases it may also be appropriate
+  for it to be sent to the Zcash Community Forum.
+* Ensure that motivation and backward compatibility have been
+  addressed, if applicable.
+* Check that the licensing terms are still acceptable for ZIPs.
+
+If an Update ZIP is accepted, it is the responsibility of the ZIP Editors
+to apply the changes it describes to any existing specifications, but
+this MUST only be done once the Status of the Update ZIP has reached at
```

Comment:
It could be useful to define "Update ZIP" in the list of ZIP categories, otherwise, the capitalization here may be confusing.